### PR TITLE
DOMA-3270 amplitude bug fixes

### PR DESCRIPTION
**Pull request by sitozzz**
- [x] update amplitude token at values.yml

### CODE REVIEW

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/user/components/UserMenu/DesktopUserMenu.tsx [hunk 1]
```diff
@@ -89,6 +89,7 @@ export const DesktopUserMenu: React.FC = () => {
                                     type='link'
                                     style={{ paddingRight: 0, color: green[6], fontSize: '12px' }}
                                     onClick={goToUserProfile}
+                                    eventName={'MenuClickSignout'}
```
Comment (AleX83Xpert) [inline apps/condo/domains/user/components/UserMenu/DesktopUserMenu.tsx:92]:
```suggestion
                                    eventName="MenuClickSignout"
```

File: apps/condo/domains/user/components/UserMenu/MobileUserMenu.tsx [hunk 2]
```diff
@@ -57,8 +57,8 @@ const ModalView: React.FC<ModalViewProps> = ({ setShowModal }: ModalViewProps) =
 
     return (
         <ModalWrapper>
-            <Button color={colors.white} onClick={onProfileItemClick}>{ProfileMessage}</Button>
-            <Button color={colors.white} onClick={onSignOutItemClick}>{SignOutMessage}</Button>
+            <Button color={colors.white} onClick={onProfileItemClick} eventName={'MenuClickProfile'}>{ProfileMessage}</Button>
+            <Button color={colors.white} onClick={onSignOutItemClick} eventName={'MenuClickSignout'}>{SignOutMessage}</Button>
```
Comment (AleX83Xpert) [inline apps/condo/domains/user/components/UserMenu/MobileUserMenu.tsx:61]:
```suggestion
            <Button color={colors.white} onClick={onProfileItemClick} eventName="MenuClickProfile">{ProfileMessage}</Button>
            <Button color={colors.white} onClick={onSignOutItemClick} eventName="MenuClickSignout">{SignOutMessage}</Button>
```

**Review by AleX83Xpert — APPROVED** [review]

File: apps/condo/domains/user/components/UserMenu/MobileUserMenu.tsx (hunk 2 above)
Comment (sitozzz) [reply apps/condo/domains/user/components/UserMenu/MobileUserMenu.tsx]: Done

File: apps/condo/domains/common/components/antd/Input.tsx [hunk 3]
```diff
@@ -18,11 +20,20 @@ const Input = (props: CustomInputProps) => {
         componentProperties['component'] = { value: restProps.value }
     }
 
-    const onChangeCallback = eventName ? getTrackingWrappedCallback(eventName, componentProperties, onChange) : onChange
+    useEffect(() => {
+        let timoutId
```
Comment (AntonAL) [inline apps/condo/domains/common/components/antd/Input.tsx:24]:
```suggestion
        let timeoutId
```

File: apps/condo/domains/common/components/antd/Input.tsx [hunk 4]
```diff
@@ -18,11 +20,20 @@ const Input = (props: CustomInputProps) => {
...
+        if (!firstRender.current && eventName && restProps.value && restProps.type !== 'hidden') {
+            timoutId = setTimeout(() => {
```
Comment (AntonAL) [inline apps/condo/domains/common/components/antd/Input.tsx:26]:
```suggestion
            timeoutId = setTimeout(() => {
```

File: apps/condo/domains/user/components/UserMenu/DesktopUserMenu.tsx [hunk 5]
```diff
@@ -89,6 +89,7 @@ export const DesktopUserMenu: React.FC = () => {
                                     type='link'
                                     style={{ paddingRight: 0, color: green[6], fontSize: '12px' }}
                                     onClick={goToUserProfile}
+                                    eventName="MenuClickSignout"
```
[the remark this answers is not in the document]
Comment (sitozzz) [reply apps/condo/domains/user/components/UserMenu/DesktopUserMenu.tsx]: Done

**Pull request merged**